compiler/lib/src/main/scala/codegen/CppWriter/PortCppWriter.scala

```diff
@@ -157,10 +157,6 @@ case class PortCppWriter (
   }
 
   private def getHppIncludes: CppDoc.Member = {
-    val systemHeaders = List(
-      "cstdio",
-      "cstring",
```

Comment:
The component implementation classes in F Prime seem to be expecting both of these headers, which aren't included in other files. Some of them use `memset()` from `cstring` or `snprintf()` from `cstdio`, so I think these might be needed here.

Reply:
OK, good catch. I think those headers are not needed here, but you're right, the component code may depend on them. Let's fix this later when we revise the component code.